superset/db_engine_specs/hive.py

```diff
@@ -98,7 +98,7 @@ def fetch_data(cls, cursor, limit: int) -> List[Tuple]:
             return []
 
     @classmethod
-    def create_table_from_csv(cls, form, table):  # pylint: disable=too-many-locals
+    def create_table_from_csv(cls, form):  # pylint: disable=too-many-locals
```

Comment:
This change will make the API for the Hive db_engine_spec diverge further from `base`. It would behoove us to keep them in sync.

Reply:
Based on my understanding of the `HiveEngineSpec.create_table_from_csv`, this actually aligns the functionality with that of `BaseEngineSpec`, as previously the Hive spec didn't create any metadata for new tables (all the others did). After this change metadata creation takes place outside of the engine spec, and works similarly for all engines, i.e. metadata is created in `database/views.py`, and only uploading of actual data is handled by the engine specs.

Reply:
Hey Ville - the implementation is fine here, I'm only referring to the removal of the `table` argument from the method signature. Sorry, I should have been more specific.

Reply:
@willbarrett I have tried to make sure the API remains compatible, i.e. both `BaseEngineSpec` ja `HiveEngineSpec` now only take `form` as additional parameters (I did forget to add the `None` return type to Hive, though). So there shouldn't be any discrepancy between them. Or did I misunderstand something?